src/huak/env/venv.rs

```diff
@@ -173,6 +173,8 @@ impl Venv {
         let name = self.name()?;
         let args = ["-m", "venv", name];
 
+        println!("Creating venv {}", self.path.display());
```

Comment:
Hmm I'm not sure this is desirable.

Reply:
I imagine stdout would be the caller's responsibility.

Reply:
Fair point. Though this will require the caller to know/remember to print that information. And also now this only prints if it actually creates the venv. If it already exists it doesn't print. So if we move this print, then the caller will have to check if the venv exists and if not then print the message and call this function, which also checks if the venv exists. And that seems convoluted to me.

Reply:
Alrighty. We're going to be moving Venv logic around a lot. So we can figure out a nicer implementation at some point anyway. There's definitely a need for the print, but maybe we can think about a less committing way to provide that.